index.html

```diff
@@ -0,0 +1,299 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>My Portfolio Figma 1</title>
+        <link rel="stylesheet" href="./style.css">
+    </head>
+</html>
+<body>
+    <main>
+        <header>
+            <span class="my-logo"><a href='#'>My Logo</a></span>
+            <nav>
+                <a href=""><img src="./assets/images/Union.png" alt="Humburger Button"></a>
+            </nav>
+        </header>
+
+        <section class="hero">
+            <h1 class="greeting">
+                I'm Sio. <br />
+                Glad to see <br />
+                you!
+            </h1>
+            <div class="job-description">
+                <p>I'm a software developer! I can help you build a product , 
+                    feature or website look through some of my work and experience!
+                    If you like what you see and have a project you need coded, 
+                    don't hesitate to contact me.
+                </p>
+            </div>
+            <h2 class="connect-link">LET'S CONNECT</h2>
+            <div class="social-media-icons">
+                <ul class="social media links">
+                    <li>
+                        <a href="#">
+                            <img src="./assets/images/twitter-logo.png" alt="twitter-logo">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
```

Comment:
Kindly make sure that your website look exactly like the Figma design.
### Your design:
![image](https://github.com/May126-sone/FIGMA-1-PORTFOLIO/assets/113283174/3dd9a67b-3f6a-4f9c-994b-ec6061f8c004)
### Figma design:
![image](https://github.com/May126-sone/FIGMA-1-PORTFOLIO/assets/113283174/8122c301-7419-41de-91b7-f48c660c5727)